### ISSUE TITLE
Is there anything Search function or sorting like Grid does?

### ISSUE THREAD

**Issue by noeljoaquin30** [kristoferjoseph/flexboxgrid#279]
Question. Is there anything Search function, sorting and Pagination like normal grid does?

**Comment by stamminator** [comment]
@noeljoaquin30 No, you're thinking of a grid/table control/widget/component. As you can see from all of my slashes, the terminology is all over the place. flexboxgrid is simply a collection of CSS classes that simplify the use of `flexbox` for displaying _any_ content (not just tabular data) in a grid structure.
